src/Controller/PanelsController.php

```diff
@@ -53,11 +53,9 @@ public function beforeFilter(Event $event)
      */
     public function beforeRender(Event $event)
     {
-        $builder = $this->viewBuilder();
-        if (!$builder->className()) {
-            $builder->layout('DebugKit.panel')
-                ->className('DebugKit.Ajax');
-        }
+        $this->viewBuilder()
+            ->layout('DebugKit.panel')
+            ->className('DebugKit.Ajax');
```

Comment:
@markstory sorry about the delay on this. I updated the builder to set the ajax view only if the request isn't JSON. As far as I can tell, only the PanelsController uses JSON.